testing/set_library_versions.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""
+This file is used by unit testing to specify library versions.
+Commands here should maintain alignment with the environment generated in 
+testing/Dockerfile.
+
+"""
+
+import json
+import sys
+import os
+
+# Read library versions from testcase library_versions.json file
+with open(os.path.join(sys.argv[1],'models','library_versions.json'),'r') as f:
+    library_versions = json.loads(f.read()) 
+
+# Checkout specified commit for each library
+for key in library_versions.keys():
+    if key == "IBPSA_COMMIT":
+        os.system('cd $MODELICAPATH/IBPSA && git checkout {}'.format(library_versions[key]))
+    elif key == "BUILDINGS_COMMIT":
+        os.system('cd $MODELICAPATH/Buildings && git checkout {}'.format(library_versions[key]))
+    elif key == "IDEAS_COMMIT":
```

Comment:
Suggest raise error if key not found with message saying so.

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/pull/223/commits/237ad498ac225b24530be09eabeba42b62a7cdc2.